systems/init/msgClient/internal/config.go

```diff
@@ -14,7 +14,7 @@ type Config struct {
 	Metrics          *uconf.Metrics    `default:"{}"`
 	Timeout          time.Duration     `default:"3s"`
 	HeathCheck       HeathCheckRoutine `default:"{}"`
-	System           string            `default:"init"`
+	System           string            `default:"subscriber"`
```

Comment:
shouldn't we be changing this value through environment variable  instead of overriding the default value of another system?

Reply:
addressed